ui/privacy_page.go

```diff
@@ -50,14 +50,15 @@ func (win *Window) PrivacyPage(common pageCommon) layout.Widget {
 func (pg *privacyPage) Layout(gtx layout.Context, c pageCommon) layout.Dimensions {
 	d := func(gtx C) D {
 		load := SubPage{
-			title:      "Privacy",
-			walletName: c.info.Wallets[*c.selectedWallet].Name,
+			title:        "Privacy",
+			walletName:   c.info.Wallets[*c.selectedWallet].Name,
+			isInfoButton: true,
+			isWalletName: true,
```

Comment:
Is this necessary? why not check if the walletName is an empty string?. Same thing applies the infoTemplate.

Reply:
Removing `isInfoButton` and `isWalletName` and doing this https://github.com/planetdecred/godcr/pull/305#discussion_r564375795 and https://github.com/planetdecred/godcr/pull/305#discussion_r564375852 leaves the info icon and a piece of the card. 

<img width="796" alt="image" src="https://user-images.githubusercontent.com/27733432/105851529-7f20c100-5fe3-11eb-935b-e6f7ece9e3ce.png">